src/odemis/acq/path.py

```diff
@@ -601,7 +602,9 @@ def _doSetPath(self, path, detector):
 
             try:
                 # move actuator
-                fmoves.append(comp.moveAbs(mv))
```

Comment:
I'd suggest more something like:
`fmoves.append(comp.moveAbs(mv), comp, mv)`.

Then, in the loop waiting for the end of the moves you can write:
`for f, comp, mv in fmoves:`

This avoids having two variables containing the same info *and*, `comp` will be the right value ;-)